Doc/whatsnew/3.12.rst

```diff
@@ -622,6 +622,18 @@ New Features
   ``__dict__`` and weakrefs with less bookkeeping,
   using less memory and with faster access.
 
+* API for performing calls using
+  :ref:`the vectorcall protocol <vectorcall>` was added to the
+  :ref:`Limited API <stable>`:
+
+  * :c:func:`PyObject_Vectorcall`
+  * :c:func:`PyObject_VectorcallMethod`
+  * :const:`Py_TPFLAGS_HAVE_VECTORCALL`
```

Comment:
```suggestion
  * :c:data:`Py_TPFLAGS_HAVE_VECTORCALL`
```

Likewise (there is a `:data:` role, which is syntactically equivalent to `var`, just not a directive)